### DISCUSSION TITLE
notebook.db二进制文件与git版本管理

### DISCUSSION THREAD

**Discussion by simaodangan** [tamlok/vnote#2347] [Q&A]
作者您好，感谢您在vnote上的工作与付出。

我和同事使用git作为笔记本的协同编辑同步工具，但是发现notebook.db是个二进制文件，不方便使用git进行同步。

请问作者notebook.db是否是必要的文件，能否加入到.gitignore文件中避免进行同步？

**Comment by feloxx** [comment]
是必要文件，里面是存索引用的。
你用sqlite打开看一下就知道了。建议加到日常的同步当中。
